scripts/CommonUtils.ps1

```diff
@@ -64,7 +64,7 @@ function Install-DotNetTool
 
 function Install-BuildTools
 {
-    Install-DotNetTool -Name ClangSharpPInvokeGenerator -Version 11.0.0-beta3
+    Install-DotNetTool -Name ClangSharpPInvokeGenerator -Version 12.0.0-beta2
```

Comment:
Please don't update this. I would need to make sure this doesn't regress anything before we update it.